javascript-practice/src/pages/login.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/_main.scss" />
+    <title>Login - Money Lover</title>
+  </head>
+  <body>
+    <!-- <div class="loader"></div> -->
+    <div class="background">
+      <div class="wrapper">
+        <img class="logo-web" src="../assets/images/logo.svg" alt="Logo web" />
+        <!-- Log in form -->
+        <form action="#" class="form" method="post">
+          <div class="form__container">
+            <h1 class="form__title">Log In</h1>
```

Comment:
Should not h1. H1 for important content for SEO

Reply:
I think every page must have an H1 heading and that is important content because apart from this one, nothing else should be more appropriate than this one.

Reply:
For all page, h1 should be a logo or branch name. For login/register page, we also have a header with logo.